include/kframework/serialization.md

```diff
@@ -53,9 +53,9 @@ Address/Hash Helpers
     rule #sender("")  => .Account
     rule #sender(STR) => #addr(#parseHexWord(Keccak256(STR))) requires STR =/=String ""
 
-    syntax Int ::= #addrFromPrivateKey ( String ) [function]
- // --------------------------------------------------------
-    rule #addrFromPrivateKey ( KEY ) => #addr( #parseHexWord( Keccak256 ( Hex2Raw( ECDSAPubKey( Hex2Raw( KEY ) ) ) ) ) )
+    syntax Int ::= #addrFromPrivateKey ( String ) [function, klabel(addrFromPrivateKey)]
+ // ------------------------------------------------------------------------------------
+    rule [addrFromPrivateKey]: #addrFromPrivateKey ( KEY ) => #addr( #parseHexWord( Keccak256 ( Hex2Raw( ECDSAPubKey( Hex2Raw( KEY ) ) ) ) ) )
```

Comment:
@ehildenb is there any difference in using the rule label and using the `klabel` tag? Also, is the klabel required anymore?

Reply:
klabel is not required, and yes there is a big difference. One is a label for referring to the production, and the other is a rule label for referring to the rule.

Reply:
No need to remove the klabel.

Reply:
You can find documentation on both in the K user manual.